src/requests/request.ts

```diff
@@ -172,7 +173,12 @@ export async function makeRequest(
 
 function handleResponseError(e: Error, url: string): void {
   let err = e;
-  if (
+  if (err.name === "AbortError") {
```

Comment:
I don't think this is the fix, on this path all errors are already wrapped into GoogleGenerativeAI Error, the issue is that on StreamReader we are not catching errors when the stream is read.

Reply:
Do we need this part changed?